daprdocs/content/en/developing-applications/crewai/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+type: docs
+title: "CrewAI"
+linkTitle: "CrewAI"
```

Comment:
@yaron2 would it make sense to make a new section in the documentation tree that aligns with Dapr Agents & Dapr for Agents?

```shell
Developing applications/
| - AI Agents/
|   | - Dapr Agents/
|       | - Introduction/
|       | - ../
|   | - Dapr for Agents/
|       | - CrewAI/
|       | - ../
|       | - LangGraph/
|       | - ../
|       | - Strands Agents/
|       | - ../
```

This would also align with the storyline of dapr agents + dapr for agents